spec/requests/request_throttling_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe 'throttling for xml requests' do
+
+  let(:limit) { 30 }
```

Comment:
Also, the limit here has to match the limit in `rack-attack.rb` for the test to work right. If I were to do the `proc` for `:limit`, both values could be pulled from config.